### ISSUE TITLE
[Question]: Why do I get a Webpack Compilation Error after setup?

### ISSUE THREAD

**Issue by SCH17044** [IBMa/equal-access#1572]
### Project

cypress accessibility checker

### Description

Hey,
I am trying to set up your accessibility checker, by using your given guide, but I am struggling a little bit.
Initially I thought that the error had something to do with the use of the old data structure, which is no longer supported since Cypress 10.0.0, but I was even more puzzled when I executed your "boilerplates" example with the given steps and also got the same error. I've tried to find a possible solution to this on the internet, but most of the forum entries are either not fitting or closed due to inactivity. Therefore, I hope that you might be able to help me.

```
Error: Webpack Compilation Error
./node_modules/cypress-accessibility-checker/src/lib/ACEngineManager.js 5:24
Module parse failed: Unexpected token (5:24)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 
| class ACEngineManager {
>     static engineLoaded = false;
|     static checker;
|     static loadOnce = null;
 @ ./node_modules/cypress-accessibility-checker/src/lib/ACCommands.js 1:28-56
 @ ./node_modules/cypress-accessibility-checker/src/commands.js
 @ ./node_modules/cypress-accessibility-checker/index.js
 @ ./cypress/e2e/accessibility-checker-example.cy.js
    at Watching.handle [as handler] (C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\@packages\server\node_modules\@cypress\webpack-preprocessor\dist\index.js:212:23)
    at C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\webpack\lib\Watching.js:99:9
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\tapable\lib\Hook.js:154:20)
    at Watching._done (C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\webpack\lib\Watching.js:98:28)
    at C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\webpack\lib\Watching.js:73:19
    at Compiler.emitRecords (C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\webpack\lib\Compiler.js:499:39)
    at C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\webpack\lib\Watching.js:54:20
    at C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\webpack\lib\Compiler.js:485:14
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\tapable\lib\Hook.js:154:20)
    at C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\webpack\lib\Compiler.js:482:27
    at C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\neo-async\async.js:2818:7
    at done (C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\neo-async\async.js:3522:9)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\tapable\lib\Hook.js:154:20)
    at C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\webpack\lib\Compiler.js:464:33
    at C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\@packages\server\node_modules\graceful-fs\graceful-fs.js:143:16
    at C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\@packages\server\node_modules\graceful-fs\graceful-fs.js:143:16
    at C:\Users\tester\AppData\Local\Cypress\Cache\12.12.0\Cypress\resources\app\node_modules\@packages\server\node_modules\graceful-fs\graceful-fs.js:61:14
    at FSReqCallback.oncomplete (node:fs:200:23)
    ```

**Comment by shunguoy** [comment]
@SCH17044 Sorry for the delay. The latest code has been deployed, please see https://github.com/IBMa/equal-access/releases/tag/3.1.58
There are multiple updates on the dependencies. Could you please try and let me know if your issue was fixed? thanks.

**Comment by SCH17044** [comment]
Hey,
thanks for your reply! I just tried and it sadly did not resolve my problem.

**Comment by shunguoy** [comment]
@SCH17044 sorry to hear the fix didn't address the issue.  we are doing further investigation on the issue.

**Comment by SCH17044** [comment]
Thanks for your effort!

**Comment by tombrunet** [comment]
@SCH17044 The 3.1.59 release was pushed today that fixes this issue and should allow you to run the current boilerplate.

**Comment by SCH17044** [comment]
@tombrunet thanks for your efforts, but I am still getting the exact same error after using release 3.1.59.
